website/docs/m-agent/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+id: overview
+title: m-agent
+sidebar_label: Overview
+---
+
+---
+
+Machine Agent, a.k.a. m-agent is a lightweight, platform-generic daemon agent that can remotely inject faults into machine scoped resources as part of the LitmusChaos Experiments.
+
+## Why do you need m-agent?
+In the context of machine-scoped chaos experiments for LitmusChaos, despite using various cloud platform specific VM agents for the execution of the chaos, there have been several limitations such as:
+- Inflexible APIs for run-command which translates to poor chaos orchestration and chaos rollback abilities.
```

Comment:
`run-command` is specific to Azure, can we term it differently, `running commands in targeted VM` is one suggestion.